lib/internal/Magento/Framework/Pricing/Render/Amount.php

```diff
@@ -79,6 +87,7 @@ public function __construct(
         RendererPool $rendererPool,
         SaleableInterface $saleableItem = null,
         PriceInterface $price = null,
+        FormatInterface $format = null,
```

Comment:
@Bartlomiejsz sorry, looks like I missed that in the previous review, newly added parameters should be at the end (after `$data`)

Reply:
@sivaschenko done